prime-router/src/main/kotlin/azure/ApiKeysFunctions.kt

```diff
@@ -31,16 +53,20 @@ class ApiKeysFunctions(private val settingsFacade: SettingsFacade = SettingsFaca
         ) request: HttpRequestMessage<String?>,
         @BindingName("organizationName") orgName: String
     ): HttpResponseMessage {
-        val claims = AuthenticatedClaims.authenticate(request)
-        if (claims == null || !claims.authorized(setOf("*.*.primeadmin", "$orgName.*.admin"))) {
-            logger.warn("User '${claims?.userName}' FAILED authorized for endpoint ${request.uri}")
-            return HttpUtilities.unauthorizedResponse(request, authenticationFailure)
-        }
-        val organization = BaseEngine.settingsProviderSingleton.findOrganization(orgName)
-            ?: return HttpUtilities.notFoundResponse(request, "No such organization: $orgName")
+        return getApiKeysForOrg(request, orgName)
+    }
 
-        val keys = organization.keys ?: emptyList()
-        return HttpUtilities.okJSONResponse(request, ApiKeysResponse(orgName, keys))
+    @FunctionName("getApiKeysV1")
```

Comment:
I added this as an example of some thoughts on versioning the API as well as a way to test out the new response object